addons/event/static/src/views/event_slot_calendar/event_slot_calendar_model.js

```diff
@@ -1,20 +1,41 @@
 import { CalendarModel } from "@web/views/calendar/calendar_model";
 import { serializeDate } from "@web/core/l10n/dates";
 
+/**
+ * Slots are created differently depending on the screen size.
+ * Desktop: Using the multi create feature.
+ * Mobile: Using the calendar quick create dialog.
+ */
 export class EventSlotCalendarModel extends CalendarModel {
 
     /**
      * @override
-     * Save slot date and hours from selected datetimes
+     * Set slot date and hours from selected datetimes.
```

Comment:
Side note, there is not anymore the idea of keeping the last duration for next slot as default?